For future reference, this is the failure:
```
C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\yams\Sources\Yams\Representer.swift:321:45: error: cannot find 'DBL_DECIMAL_DIG' in scope
319 |         // Since `NumberFormatter` creates a string with insufficient precision for Decode,
320 |         // it uses with `String(format:...)`
321 |         let string = String(format: "%.*g", DBL_DECIMAL_DIG, self)
    |                                             `- error: cannot find 'DBL_DECIMAL_DIG' in scope
322 |         // "%*.g" does not use scientific notation if the exponent is less than â€“4.
323 |         // So fallback to using `NumberFormatter` if string does not uses scientific notation.
```

--- a/stdlib/public/Platform/ucrt.modulemap
+++ b/stdlib/public/Platform/ucrt.modulemap
@@ -25,19 +25,6 @@ module _complex [system] [no_undeclared_includes] {
   export *
 }
 
-module _fenv [system] [no_undeclared_includes] {
-  use corecrt
-  use _float
-  header "fenv.h"
-  export *
-}
-
-module _float [system] [no_undeclared_includes] {
-  use corecrt
-  header "float.h"
-  export *
-}
-
 module _stddef [system] [no_undeclared_includes] {
   header "stddef.h"
   export *
@@ -54,8 +41,6 @@ module ucrt [system] {
 
   module C {
     export _complex
-    export _fenv
-    export _float
     export _stddef
     export _stdlib
 
@@ -66,6 +51,16 @@ module ucrt [system] {
 
     module errno {
       header "errno.h"
+      export *
+    }
+
+    module fenv {
+      header "fenv.h"
+      export *
+    }
+
+    module float {
+      header "float.h"
       export *
     }
 

